app/lib/linters/spectral.rb

```diff
@@ -1,18 +1,24 @@
 module Linters
+  class SpectralLinterError < StandardError; end
```

Comment:
I wouldn't say its cleaner, it seems the guy in the article just prefers it. I'm not sure there's enough to it to warrant a change. I've seen both ways used in quite a few projects and I've always just preferred inheriting rather than assigning an anonymous class.